README.md

```diff
@@ -321,6 +321,7 @@ described in [RFC 8032]
  * [ffmpeg](https://github.com/spider-gazelle/ffmpeg) - FFmpeg bindings that works with StumpyPNG to extract frames
  * [Pluto](https://github.com/phenopolis/pluto) - A fast and convenient image processing library
  * [stumpy_png](https://github.com/stumpycr/stumpy_png) - Read and write PNG images
+ * [CrystalVips](https://github.com/naqvis/crystal-vips) - Crystal bindings to [libvips](https://github.com/libvips/libvips) image processing library
```

Comment:
If this lib is purely bindings to `libvips`, it should probably go under the `C Bindings` section. But if it's bindings + a Crystal API on top of it, probably should update the desc to point that out. E.g. something like: 

> Image processing library based on [libvips](https://github.com/libvips/libvips)

To denote its more than just bindings.